doc/en/skipping.rst

```diff
@@ -301,7 +288,7 @@ on a particular platform:
 
 .. code-block:: python
 
-    @pytest.mark.xfail(sys.version_info >= (3, 6), reason="python3.6 api changes")
+    @pytest.mark.xfail(reason="known parser issue")
```

Comment:
This example does not match the previous sentence anymore.

Reply:
Thanks, https://github.com/pytest-dev/pytest/pull/6980